packages/vitest/src/node/reporters/default.ts

```diff
@@ -1,18 +1,22 @@
 import type { Vitest } from '../core'
 import type { TestSpecification } from '../spec'
 import type { BaseOptions } from './base'
+import type { ReporterColors } from './renderers/utils'
 import type { ReportedHookContext, TestCase, TestModule } from './reported-tasks'
 import { BaseReporter } from './base'
 import { SummaryReporter } from './summary'
 
 export interface DefaultReporterOptions extends BaseOptions {
+  colors?: ReporterColors
   summary?: boolean
 }
 
 export class DefaultReporter extends BaseReporter {
   private options: DefaultReporterOptions
   private summary?: SummaryReporter
 
+  protected colors: DefaultReporterOptions['colors']
```

Comment:
1. Colors should also be inherited by UI, so limiting it to reporters won't work
2. Since UI also requires colors, we can't use ANSII or functions, it should be hard coded names

Reply:
Thank you for the feedback, I'll work on another solution.